server/txSender/txSender_test.go

```diff
@@ -115,7 +126,7 @@ func TestTxSender_SendTxs(t *testing.T) {
 			require.Equal(t, &transaction.FrontendTransaction{
 				Nonce:    0,
 				Value:    "0",
-				Receiver: args.SCBridgeAddress,
+				Receiver: tx.Receiver,
```

Comment:
Not ok, we should check that the receiver is the expected one. Please follow the same pattern as used here:
```
Data: expectedTxsData[expectedDataIdx],
```

Reply:
Where:
- first receiver should be: `scMultisigAddress`
- second and third should be: `scEsdtSafeAddress`